ubuntu-rsp-rsc/README.md

```diff
@@ -10,10 +10,10 @@ This directory contains everything you need to run two virtual machines, one for
 
 ### Download Ubuntu 16.04 Box
 
-Retrieve a pre-built [Ubuntu image from Vagrant Cloud](https://app.vagrantup.com/ubuntu/boxes/xenial64):
+Retrieve a pre-built [Ubuntu image from Vagrant Cloud](https://app.vagrantup.com/bento/boxes/ubuntu-16.04):
 
 ```
-vagrant box add ubuntu/xenial64
+vagrant box add bento/ubuntu-16.04
```

Comment:
All boxes actually using box from `bento`, not `ubuntu`, figured I'd harmonize documentation.